modules/simple_forms_api/app/models/simple_forms_api/vha_10_10d.rb

```diff
@@ -18,17 +18,21 @@ def metadata
         'zipCode' => @data.dig('veteran', 'address', 'postal_code') || '00000',
         'source' => 'VA Platform Digital Forms',
         'docType' => @data['form_number'],
-        'businessLine' => 'CMP'
+        'businessLine' => 'CMP',
+        'ssn_or_tin' => @data.dig('veteran', 'ssn_or_tin')
       }
     end
 
     def handle_attachments(file_path)
+      ssn_or_tin = @data.dig('veteran', 'ssn_or_tin')
+      file_path_ssn = file_path.gsub('vha_10_10d-tmp', "#{ssn_or_tin}_vha_10_10d-tmp")
+      File.rename(file_path, file_path_ssn)
       attachments = get_attachments
-      file_paths = [file_path]
+      file_paths = [file_path_ssn]
 
       if attachments.count.positive?
         attachments.each_with_index do |attachment, index|
-          new_file_name = "vha_10_10d-tmp#{index + 1}.pdf"
+          new_file_name = "#{ssn_or_tin}_vha_10_10d-tmp#{index + 1}.pdf"
           new_file_path = File.join(File.dirname(attachment), new_file_name)
```

Comment:
@cloudmagic80 I thought these changes were going to be for all the IVC forms not just 10-10D?

Reply:
That is true. it will be for all IVC forms. Just for this commit, it affects 1010d.